xml/homepage.xml

```diff
@@ -0,0 +1,80 @@
+<!--
+  Copyright (C) Nginx, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../dtd/article.dtd">
+
+<article name="nginx"
+         link="/"
+         lang="en"
+         rev="1">
+
+<section>
+
+<para>
+nginx ("<i>engine x</i>") is an HTTP web server, reverse proxy,
+content cache, load balancer,
+TCP/UDP proxy server,
+and mail proxy server.
+Originally written by <link url="http://sysoev.ru/en/">Igor Sysoev</link>
+and distributed under the
+<link url="../LICENSE">2-clause BSD License</link>.
+Enterprise distributions, commercial support and training are
+<link url="enterprise.html">available from F5, Inc.</link>
```

Comment:
https://nginx.org/enterprise.html is 404 for me.

Reply:
Thank you! Submitted fix as https://github.com/nginx/nginx.org/pull/86

Reply:
Fixed, thanks. The update should appear on the site shortly.